tcms/testplans/static/testplans/js/get.js

```diff
@@ -138,15 +139,20 @@ function attachEvents(testCases, testPlanId, permissions) {
             const testCaseId = getCaseIdFromEvent(ev);
 
             jsonRPC('TestCase.update', [testCaseId, {'priority': ev.target.dataset.id}], function(result) {
-                // todo update tc row data
+                const testCaseRow = $(ev.target).closest('.js-testcase-row');
+                testCaseRow.fadeOut(500).fadeIn(500);
+                testCaseRow.find('.js-test-case-priority').html(ev.target.innerText);
             });
         });
 
         $('.js-test-case-menu-status').click(function(ev) {
             const testCaseId = getCaseIdFromEvent(ev);
 
             jsonRPC('TestCase.update', [testCaseId, {'case_status': ev.target.dataset.id}], function(result) {
-                // todo update tc row data
+                //todo: refactor when testcase_status is replaced with boolean flag
+                if (result.case_status_id !== 2) {
+                    $(ev.target).closest(`[data-testcase-pk=${testCaseId}]`).remove();
```

Comment:
That should probably also update the internal data structure too b/c of the status change

Reply:
Both here and when deleting I played around with .fadeOut() and then removing the element but doesn't seem to work. Adding the remove as a callback to .fadeOut does the trick.

Similar behavior can be observed in all of the other places. For example updating the priority first updates the HTML and then the animations happen. Not a big deal but if we're going to change because of fadeOut+remove I say change everything else for consistency.